Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -583,11 +594,10 @@ def get_field_value_list(self, field_id):
             list_id = res['RequestedObject']['RelatedValuesListId']
             values_list_res = self.do_request('GET', f'{API_ENDPOINT}/core/system/valueslistvalue/valueslist/{list_id}')
             if values_list_res.get('RequestedObject') and values_list_res.get('IsSuccessful'):
-                values_list = []
-                for value in values_list_res['RequestedObject'].get('Children'):
-                    values_list.append({'Id': value['Data']['Id'],
-                                        'Name': value['Data']['Name'],
-                                        'IsSelectable': value['Data']['IsSelectable']})
+                values_list: List[Dict[str, Any]] = []
+                depth = arg_to_number(args.get('depth', '0'))
```

Comment:
why do you pass all of args if you use only depth? can we just pass depth?

Reply:
Yeah, I'll change it